src/server/session.ts

```diff
@@ -96,8 +96,9 @@ module ts.server {
         export var Reload = "reload";
         export var Rename = "rename";
         export var Saveto = "saveto";
-        export var SignatureHelp = "signatureHelp";        
-        export var TypeDefinition = "typeDefinition";        
+        export var SignatureHelp = "signatureHelp";
+        export var TypeDefinition = "typeDefinition";
+        export var ProjectInfo = "projectInfo";
         export var Unknown = "unknown";

```

Comment:
These are constants, they should be communicated as such :)

Also, it is generally good practice to mark things immutable when possible anyhow.